tests/conftest.py

```diff
@@ -80,7 +81,8 @@ def args(  # noqa: PLR0913
     """Return parsed CLI arguments for the adapter/vLLM."""
     # avoid parsing pytest arguments as vllm/vllm_tgis_adapter arguments
 
-    extra_args: list[str] = []
```

Comment:
What's this for? Unless I'm not seeing something, I don't think this should be changed